ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -31,7 +31,7 @@
     private static final By CONFIRM = By.cssSelector("#quick-visit-creation-dialog .confirm");
     private static final By STARTED_AT = By.className("active-visit-started-at-message");
     private static final By VISIT_NOTE = By.id("referenceapplication.realTime.simpleVisitNote");
-    private static final By PATIENT_CAPTURE_VITALS = By.id("referenceapplication.realTime.vitals");
+    private static final By PATIENT_CAPTURE_VITALS = By.linkText("Capture Vitals");
```

Comment:
replace with an element selector
`By.cssSelector("a[id='referenceapplication.realTime.vitals']")`

Reply:
Thanks @kaweesi . Caught this in the updated commit

Reply:
After simulating the test, the element PATIENT_CAPTURE_VITALS is retrievable by the existing attribute.

Reply:
After simulating the test, the element PATIENT_CAPTURE_VITALS is retrievable by the existing attribute.

Reply:
what did u finally do with this?

Reply:
The element is retrievable by the existing set attribute. Changing to _cssSelector_ could be alternative to retrieving the element. @k-joseph should I go for _cssSelector_

Reply:
This is @kdaud  once again! 
After simulating the behaviour of the element (PATIENT_CAPTURE_VITALS) a couple of times, I observed that the element fails to be retrieved by selenium API with the existing attribute. Took time to use different look ups via is (i_d,cssSelector, xpath, and others_) and realised the element only fails with chrome headless =true. Simulated a use case when retrieving the element via By.cssSelector("a[id='referenceapplication.realTime.vitals']")  and similar failure occurs as well. The best look up for this element is via _linkText()_ which passes on both chrome and firefox instances. cc: @k-joseph